CMS/static/scss/crc.scss

```diff
@@ -234,7 +234,16 @@ optgroup {
 
 table {
   border-collapse: collapse;
-  border-spacing: 0
+  border-spacing: 0;
+  a {
+ color: #3471B7 ;
+    text-decoration: underline;
+
+    &:hover, &:focus {
+      color: #646E9B;
+      text-decoration: underline;
+    }
+  }
```

Comment:
please don't add new code to `crc.scss`
add this to `global.scss` - it will take precedence when the css is applied